macros/base/start_print.cfg

```diff
@@ -393,3 +482,54 @@ gcode:
             ADAPTIVE_BED_MESH SIZE={FL_SIZE}
         {% endif %}
     {% endif %}
+
+
+[gcode_macro _MODULE_EXTRUDER_PREHEATING]
+gcode:
+    # Preheat the nozzle to safe probing temperature.
+    {% set safe_extruder_temp = printer["gcode_macro _USER_VARIABLES"].safe_extruder_temp|float %}
+    {% set status_leds_enabled = printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
+    {% set verbose = printer["gcode_macro _USER_VARIABLES"].verbose %}
+
+    {% if status_leds_enabled %}
+        STATUS_LEDS COLOR="HEATING"
+    {% endif %}
+    {% if verbose %}
+        RESPOND MSG="Pre-heating the nozzle to a safe temperature..."
+    {% endif %}
+
+    M109 S{safe_extruder_temp}
```

Comment:
This is a problem with low inertia hotend.
As we ran the PID at high temperatures and the algorithm is not optimal, at low temperatures the measurement oscillates wildly and we get stuck here for a few minutes until the temperature luckily stabilizes.

Reply:
Yes this hacky patch was done for this purpose :)